docs/src/add-services/redis.md

```diff
@@ -189,62 +184,84 @@ const value = await client.get('x'); // returns 42
 
 {{< /codetabs >}}
 
+## Relationship
+
+After you've [configured your Redis service](#usage-example),
+you can retrieve information such as the credentials and port you need to connect to your Redis service.
+
+To do so, you can access the [`PLATFORM_RELATIONSHIPS` environment variable](../development/variables/use-variables.md#use-platformsh-provided-variables) directly [in your app](../../development/variables/use-variables.md#access-variables-in-your-app),
+for example using a Platform.sh configuration reader.
+
+The format exposed in the `PLATFORM_RELATIONSHIPS` environment variable is identical
+for [ephemeral](#ephemeral-redis) and [persistent](#persistent-redis) Redis:
+
+{{< relationship "redis" >}}
+
 ## Eviction policy
 
-On the Ephemeral `redis` service it's also possible to select the key eviction policy.
-That controls how Redis behaves when it runs out of memory for cached items and needs to clear old items to make room.
+When the [ephemeral Redis](#ephemeral-redis) reaches its memory limit,
+it triggers a cache cleanup.
+To customize those cache cleanups, set up an eviction policy:
 
-```yaml
+```yaml {location=".platform.app.yaml"}
+web:
 cache:
     type: redis:5.0
     configuration:
         maxmemory_policy: allkeys-lru
```

Comment:
:clipboard: With the added key (which was good to notice), the indentation of the others should change.
```suggestion
web:
    cache:
        type: redis:5.0
        configuration:
            maxmemory_policy: allkeys-lru
```